packages/flutter/README.md

```diff
@@ -37,6 +39,17 @@ This library gives you access to the powerful Parse Server backend from your Flu
 
 ---
 
+## Compatibility
+
+The Parse Flutter SDK is continuously tested with the most recent release of the Flutter framework to ensure compatibility. To give developers time to upgrade their app to the newest Flutter framework, previous Flutter framework releases are supported for at least 1 year after their [release date](https://docs.flutter.dev/release/archive?tab=linux). The Parse Flutter SDK depends on the Parse Dart SDK which may require a higher Dart framework version than the Flutter framework version, in which case the Flutter framework version cannot be supported even though its release date may have been less than a year ago.
+
+| Version      | End of Support | Compatible                                   |
+|--------------|----------------|----------------------------------------------|
+| Flutter 3.10 | May 2024       | ✅ Yes                                        |
+| Flutter 3.7  | Apr 2024       | ✅ Yes                                        |
+| Flutter 3.3  | Jan 2024       | ✅ Yes                                        |
```

Comment:
We will change this letter before we publish version 5.0.0, right? because Flutter 3.3 probably will not work

The same for Dart 2.18 & Dart 2.17 probably will not work

Reply:
Yes, we can change this as needed before release.

The current Flutter table looks like this:
<img width="599" alt="image" src="https://github.com/parse-community/Parse-SDK-Flutter/assets/5673677/195a5048-7bc5-4013-b4ec-d7e56849a4f9">

Because the CI passes only for Flutter 3.7 so far :-/

Reply:
Yes, in current situation is accurate